doc/changes/changes_1.0.0.md

```diff
@@ -63,6 +63,7 @@ dictionary to the `http_transport.ImportQuery` and `http_transport.ExportQuery`
 * #179: Modified `ExaConnection` so that default is encryption with strict certification verification
 * #190: Added nox task to run examples
 * #194: Modified `ExaSQLThread` to include PUBLIC KEY in IMPORT and EXPORT queries for Exasol DB version 8.32.0 and onwards
+* #214: Added support to EXPORT and IMPORT from Exasol to Polars DataFrames
```

Comment:
Do we want polars highlighted more - like in summary or announcements? Do we do explicit contribution mentions (didn't find reference for)?